src/Components/Components.sln

```diff
@@ -0,0 +1,509 @@
+﻿
```

Comment:
Did you mean to add this .sln file? I'm guessing not.

Reply:
No, it's the freaking vs code extension that creates this garbage automatically